doc/esmvalcore/preprocessor.rst

```diff
@@ -487,8 +824,49 @@ will guarantee homogeneous input for the diagnostics.
 
 .. note::
    Conversion is only supported between compatible units! In other
-   words, converting temperature units from `degC` to `Kelvin` works
+   words, converting temperature units from ``degC`` to ``Kelvin`` works
    fine, changing precipitation units from a rate based unit to an
    amount based unit is not supported at the moment.
 
 See also :func:`esmvalcore.preprocessor.convert_units`.
+
+
+.. _MemoryUse:
+
+Information on maximum memory required
+======================================
+In the most general case, we can set upper limits on the maximum memory the
+anlysis will require:
+
+
+``Ms = (R + N) x F_eff - F_eff`` - when no multimodel analysis is performed;
+
+``Mm = (2R + N) x F_eff - 2F_eff`` - when multimodel analysis is performed;
+
+where
+
+* ``Ms``: maximum memory for non-multimodel module
+* ``Mm``: maximum memory for multimodel module
+* ``R``: computational efficiency of module; `R` is typically 2-3
+* ``N``: number of datasets
+* ``F_eff``: average size of data per dataset where ``F_eff = e x f x F``
+  where ``e`` is the factor that describes how lazy the data is (``e = 1`` for
+  fully realized data) and ``f`` describes how much the data was shrunk by the
+  immediately previous module, e.g. time extraction, area selection or level
+  extraction; note that for fix_data ``f`` relates only to the time extraction,
+  if data is exact in time (no time selection) ``f = 1`` for fix_data so for
+  cases when we deal with a lot of datasets ``R + N \approx N``, data is fully
+  realized, assuming an average size of 1.5GB for 10 years of `3D` netCDF data,
+  ``N`` datasets will require:
+
+
+``Ms = 1.5 x (N - 1)`` GB
+
+``Mm = 1.5 x (N - 2)`` GB
+
+As a thumb rule, the maximum required memory at a certain time, when meeding
```

Comment:
Rather be "As a rule of thumb, ..." instead of "As a thumb rule, ..."?

Reply:
:+1: (thumb)